### DISCUSSION TITLE
Ask Question

### DISCUSSION THREAD

**Discussion by Hisyamnm** [opengeos/opengeos.github.io#3] [General]
Hello, My name is Hisyam. I have question about WhiteBoxTools. My Arcgis is 10.8 Version, but why I can't launch the WBT and all the tools can't appear. Can you help me? Thanks in advance.

**Comment by giswqs** [comment]
Please open an issue https://github.com/opengeos/WhiteboxTools-ArcGIS/issues and provide screenshot of the error.
